### PR TITLE
Dependencies readme

### PR DESCRIPTION
**Pull request by SamJaarsma**
added dependencies and usage to the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,4 +38,45 @@ It includes a `MetadataInterceptor` that adds the `recipient` as grpc metadata t
...
 ## Versioning
 This project uses semantic versioning and releases a new version regularly upon adding support of a newer release of the camino messenger protocol.
-Version names do not have to match identically with the camino messenger protocol version, but they should be in sync.
\ No newline at end of file
+Version names do not have to match identically with the camino messenger protocol version, but they should be in sync.
+
+
+## Dependencies
+
+This chapter outlines the key dependencies required to build, package, and run the project. Ensure all dependencies are installed and properly configured before proceeding with the project setup.
```
Comment (mo-c4t) [inline README.md]: you don't need to install dependencies here, only maven and jre need to be installed, but the others they are written in pom.xml and you don't need to install them on your local machine

File: README.md [hunk 2]
```diff
@@ -38,4 +38,45 @@ It includes a `MetadataInterceptor` that adds the `recipient` as grpc metadata t
...
+
+## Usage
+
+- **mvn package**
+  - After setting up Maven, the next step is to package the project. This step compiles the source code, runs the tests, and packages the compiled code into a JAR file.
+  - Use the command `mvn package` to execute this step. This command creates a JAR file in the `target/` directory, which will be used in the following steps.
+
+- **Run the Packaged JAR**
+  - Once the JAR file is created, you can run the project using the Java command.
+  - Use the following command to run the packaged application: `java -jar target/cmb-plugin-0.0.1-SNAPSHOT.jar`.
```
Comment (mo-c4t) [inline README.md]: maybe we should add a note here that 0.0.1-SNAPSHOT is the package version specified in pom.xml , so if the user changes the version in pom.xml the package name will change.

**Review by mo-c4t — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (SamJaarsma) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (SamJaarsma) [reply README.md]: added

**Review by mo-c4t — APPROVED** [review]

**Pull request merged**